packages/SwingSet/src/kernel/vatManager/manager-subprocess-xsnap.js

```diff
@@ -75,7 +74,7 @@ export function makeXsSubprocessFactory({
       vatSyscallHandler,
       true,
       compareSyscalls,
-      useTranscript,
+      true,
```

Comment:
Could we do an `assert(managerOptions.useTranscript)` instead of putting a literal `true` in this argument list? My concern is that from here it isn't obvious what the `true` is doing. I had to look up what the existing `true` is for (`workerCanBlock`). And if somebody thinks they can set `useTranscript: false`, I'd rather throw an error than silently do something surprising.

Maybe this just means `makeManagerKit` has too many parameters, and switching to a named-property bag of stuff would make the call site more legible. Probably a cleanup to do some other time.

Reply:
OK.   All the non-local worker types now pass `useTranscript` but throw if it's not true.